docs/user-guide/mobile-sdk.md

```diff
@@ -0,0 +1,37 @@
+All SDK libraries available as pre-built packages on the GitHub [release page](https://github.com/mysteriumnetwork/node/releases).
+It can be downloaded and installed manually with any preferred way.
+
+But we also support several standard ways for installing SDK using dependency managers.
```

Comment:
We also support several standard ways of installing the SDK using dependency managers.